src/client/app/types/redux/units.ts

```diff
@@ -46,8 +46,10 @@ export interface UnitEditData {
 	displayable: DisplayableType;
 	preferredDisplay: boolean;
 	note: string;
-	defaultMeterMinimumValue:-999999999999,
-	defaultMeterMaximumValue:999999999999,
+	defaultMeterMinimumValue: number;
```

Comment:
This whole section UnitEditData can be commented out as it is not used elsewhere in the application